packages/ui/src/lib/tooltip/Tooltip.svelte

```diff
@@ -74,6 +75,15 @@ export let left = false;
     class:top-right="{topRight}"
     class:bottom-left="{bottomLeft}"
     class:bottom-right="{bottomRight}">
-    <slot name="tip" />
+    {#if tip}
+      <div class="inline-block py-2 px-4 rounded-md bg-charcoal-800 text-xs text-white" aria-label="tooltip">
+        {tip}
+      </div>
+    {/if}
+    {#if $$slots.tip && !tip}
+      <div class="inline-block py-2 px-4 rounded-md bg-charcoal-800 text-xs text-white" aria-label="tooltip">
```

Comment:
Should we remove this wrapper from the slot? I expect that if one wants to use the slot, he's going to also add different classes and this could cause some interference

Reply:
I saw that's how it was before, but I thought it was odd for client to have to know/be required to use something like inline-block and aria-label, plus to allow completely custom popup with no styling from the library. Do you see cases where you think we should allow completely custom (e.g. square) popups?

When I rebased I removed the padding, and I think this is the sweet spot - the client doesn't need to know about inline/aria, is forced to paint within a rounded window (our tooltip style), and picks up our default styling for bg/text, but these can be overridden easily.

Reply:
@lstocchi please take another look